content/influxdb/v2.0/reference/cli/influx/v1/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: influx v1
+description: >
+  The `influx v1`
+menu:
+  influxdb_2_0_ref:
+    name: influx v1
+    parent: influx
+weight: 101
+influxdb/v2.0/tags: []
+---
```

Comment:
Do we have examples to show when would folks use these commands?

For example, would they use these commands to reconfigure tools they used with 1.x, like dashboards?

Reply:
Let's link to our 1.x compatibility API docs